setup/steps/install_milk_and_cacao.sh

```diff
@@ -72,5 +72,6 @@ else
   make_on_data_array "cacao-${MAGAOX_ROLE,,}" /opt/MagAOX
   sudo ln -sf "/opt/MagAOX/cacao-${MAGAOX_ROLE,,}" /opt/MagAOX/cacao
 fi
+grep -q '^xsup:' /etc/passwd || exit 0
```

Comment:
I think I'd rather have this be an if statement for future maintainability (i.e. so we could add something after without being surprised)

Reply:
Like this?
```
if grep -q '^xsup:' /etc/passwd ; then
  ### do xsup-related stuff
fi
```